examples/carla/carla_simulator.py

```diff
@@ -504,11 +581,15 @@ def _to_transform(poses: list) -> Tuple[list, list]:
         return (t, speed)
 
     def get_entrance(self, spawn_points):
+        """
+        Filters spawn points to leave those that are entrances
+        into the supported area.
+        """
         entrance = []
         for sp in spawn_points:
             distance = math.sqrt(
-                ((sp.location.x - self.cfg.roi_center.x) ** 2)
-                + ((sp.location.y - self.cfg.roi_center.y) ** 2)
+                ((sp.location.x - self.roi_center.x) ** 2)
+                + ((sp.location.y - self.roi_center.y) ** 2)
             )
             if (
                 self.proximity_threshold - self.cfg.slack
```

Comment:
Can we handle the exits and entrances using `is_in_supported_area`? It would be very illustrative for users. We can adjust those boundaries server side if the ones defined there aren't good.

Reply:
We can but currently, the supported_area on the server side is quite bigger than what we have on the client side, and since the z coordinate of the agents is not changed the cars may fly or go into the ground!

Reply:
I opened an issue to address this later.